networks/_modules/docker/geth.tf

```diff
@@ -170,6 +170,10 @@ ARGS="--identity Node${count.index + 1} \
 %{if var.privacy_marker_transactions~}
   --privacymarker.enable \
 %{endif~}
+%{if var.private_trie_cache~}
+%{else}
+  --privatetriecache.enable=false \
```

Comment:
I'm not sure you need the `=false`

Reply:
right, that logic was when it was enabled by default